src/libc/stdio.rs

```diff
@@ -87,6 +88,13 @@ fn fgetc(env: &mut Environment, file_ptr: MutPtr<FILE>) -> i32 {
     }
 }
 
+fn fputs(env: &mut Environment, str: ConstPtr<u8>, stream: MutPtr<FILE>) -> i32 {
+    let str_len = strlen(env, str);
+    fwrite(env, str.cast(), str_len, 1, stream)
```

Comment:
This actually has the wrong behaviour on error (doesn't set errno or return EOF), maybe add a TODO comment?

Reply:
Done :)